administrator/components/com_associations/views/association/tmpl/edit.php

```diff
@@ -59,8 +57,29 @@
     <div class="outer-panel">
         <div class="inner-panel right-panel">
             <h3><?php echo JText::_('COM_ASSOCIATIONS_ASSOCIATED_ITEM'); ?></h3>
-            <iframe src="<?php echo JRoute::_('index.php?option='. $associatedComponent . '&view=' . $associatedView . '&layout=modal&tmpl=component&task=' . $associatedView . '.edit&id=2' ); ?>" name="<?php echo JText::_('COM_ASSOCIATIONS_TITLE_MODAL'); ?>" height="100%" width="400px" scrolling="no"></iframe>
+            <select id="ref-language" name="ref-language">
+                <?php echo JHtml::_('select.options', JHtml::_('contentlanguage.existing', false, true), 'value', 'text'); ?>
+            </select>
+            <button onclick="return triggerSave();" class="btn btn-small btn-success">
+                <span class="icon-apply icon-white"></span>Save
+            </button>
+            <iframe id="target-association" name="target-association" src="<?php echo JRoute::_('index.php?option='. $associatedComponent . '&view=' . $associatedView . '&layout=modal&tmpl=component&task=' . $associatedView . '.edit&id=1' ); ?>" name="<?php echo JText::_('COM_ASSOCIATIONS_TITLE_MODAL'); ?>" height="100%" width="400px" scrolling="no"></iframe>
         </div>
     </div>
 </div>
 </form>
+
+<script>
```

Comment:
use joomla API to add inline scripts and move it to the beggining of the file.

`$app->getDocument()->addScriptDeclaration('xxxxxxxxxx');`